phpcs.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0"?>
```

Comment:
This file should be called `phpcs.xml.dist`, and `phpcs.xml` should be added to a `.gitignore`.